readme.md

```diff
@@ -704,7 +704,8 @@
 | [Theatre.js](https://www.theatrejs.com/)| Theatre.js is a JavaScript animation library with a GUI. It animates the DOM, WebGL, and any other JavaScript variable. |
 | [splidejs](https://splidejs.com/)| Splide is a lightweight, flexible and accessible slider/carousel written in TypeScript. No dependencies, no Lighthouse errors. |
 | [nice-waves](https://gvguy.github.io/nice-waves/)| Beautiful animated waves |
-| [Vanta.js](https://www.vantajs.com/)| Animated website backgrounds in a few lines of code. |
+| [Vanta.js](https://www.vantajs.com/)| Animated website backgrounds in a few lines of code.
```

Comment:
Please revert this line, and change title of the pull request to follow our convention